rules/do-you-use-source-control-for-powerapps/rule.md

```diff
@@ -0,0 +1,89 @@
+---
+type: rule
+title: Do you use source control for your powerApps?
+uri: do-you-use-source-control-for-powerapps
```

Comment:
Hi @leahy268 

1. Can you remove the "do-you" as per https://github.com/SSWConsulting/SSW.Rules.Content/wiki/How-to-Create-Rules 

Note: The folder name must also be updated

Reply:
Done